k8s/openebs-operator.yaml

```diff
@@ -77,6 +77,14 @@ spec:
         ports:
         - containerPort: 5656
         env:
+        # Enables this service to connect to K8s based on this config
+        # Ignored if empty
+        - name: OPENEBS_IO_KUBE_CONFIG
```

Comment:
We should probably comment these values. And also provide valid values for users to replace, similar to how default conf files in /etc/ are shown. 

Let the same PR also include the changes required for the openebs provisioner.